### PR TITLE
templates: add range facet

### PR DESCRIPTION
**Pull request by diegodelemos**
Addresses inveniosoftware/invenio-app-ils#21

### CODE REVIEW

File: invenio_search_ui/static/templates/invenio_search_ui/range.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="cds-search-facets bg-w pt-30" ng-show="vm.invenioSearchResults.aggregations.years.buckets.length > 0">
+    <div class="cds-border ba pa-10 b-r-3">
+        <h2 class="text-capitalize f8 mt-5">
```
Comment (drjova) [inline invenio_search_ui/static/templates/invenio_search_ui/range.html]: we can remove this template :P we are using the on from ``invenio-search-js`` :)
```
SEARCH_UI_JSTEMPLATE_RANGE = 'node_modules/invenio-search-js/dist/templates/range.html'
```

**Comment by drjova** [comment]
ping @diegodelemos :P

**Comment by diegodelemos** [comment]
This makes the error disappear but still needs work since if there are two issues.
* The block for `range.html` template gets rendered even if there are no results.
* When I provide this config in `invenio-app-ils` I get two new facets on the left.
```
@@ -128,6 +128,13 @@ RECORDS_REST_FACETS = dict(
                     field='main_entry_personal_name.affiliation'
                 )
             ),
+            years=dict(
+                date_histogram=dict(
+                    field='_created',
+                    interval='year',
+                    format='yyyy'
+                )
+            )
         ),
         post_filters=dict(
             identifier=terms_filter(
@@ -140,6 +147,11 @@ RECORDS_REST_FACETS = dict(
             affiliation=terms_filter(
                 'main_entry_personal_name.affiliation'
             ),
+            years=range_filter(
+                '_created',
+                format='yyyy',
+                end_date_math='/y'
+            ),
         )
     )
 )
``` 
screen shot 2017-08-01 at 2 33 59 pm

**Comment by diegodelemos** [comment]
ready @drjova

**Pull request merged**